docs/templates/plugin.rst.j2

```diff
@@ -220,15 +222,13 @@ Status
 This module is flagged as **@{cur_state}@** which means that @{module_states[cur_state]}@.
 {% endfor %}
 {% endif %}
+{% if metadata.supported_by in ('core', 'network') %}
 
-{% if metadata.supported_by %}
+Maintenance Info
+~~~~~~~~~~~~~~~~
 
-Support Level
-~~~~~~~~~~~~~
-
-This module is maintained by @{ support[metadata.supported_by] }@
-
-For more information on what this means please read :doc:`modules_support`
+For more information about this module’s supportability from Red Hat, please
```

Comment:
Consider revising to "For more information about Red Hat's support for this module..."